src/Formatter/Header.php

```diff
@@ -4,34 +4,16 @@
 
 class Header
 {
-    /** @var Cron */
-    private $cron;
-
-    /** @var string */
-    private $path;
-
-    /** @var string */
-    private $mailTo;
-
-    /** @var string */
-    private $home;
-
-    /** @var string */
-    private $shell;
-
-    /** @var string */
-    private $contentType;
-
-    /** @var string */
-    private $encoding;
-
-    /** @var string */
-    private $timezone;
-
-    public function __construct(Cron $cron)
-    {
-        $this->cron = $cron;
-    }
+    private string $path;
```

Comment:
Path could be null too from what I see?
```suggestion
    private ?string $path = null;
```